cricPredict/cricPredict/settings.py

```diff
@@ -137,11 +133,24 @@
 }
 
 REST_FRAMEWORK = {
-        'DEFAULT_PERMISSION_CLASSES': [
-            'rest_framework.permissions.IsAuthenticated',
-            'rest_framework.permissions.IsAuthenticatedOrReadOnly',
-        ],
-        'DEFAULT_AUTHENTICATION_CLASSES': (
-            'rest_framework.authentication.BasicAuthentication',
-        )
-}
\ No newline at end of file
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.IsAuthenticated',
+        'rest_framework.permissions.AllowAny',
+    ],
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        "rest_framework.authentication.SessionAuthentication",
```

Comment:
we should remove `SessionAuthentication` to disallow any old cookie-based login, you can verify this if you login into admin and then visit any API endpoint inside the browser, it'll show you DRF's browsable API and show you that the user is logged in(the one you logged in from admin).